src/styles/blocks/card.scss

```diff
@@ -0,0 +1,75 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid $cart-border-color;
+  border-radius: 5px;
+  background-color: $cart-bg-color;
+  padding: $card-padding;
+
+  &__product-img {
+    display: flex;
+    justify-content: center;
+    padding: $product-img-padding 0;
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    font-weight: 500;
+    line-height: 18px;
+    color: $page-text-color;
+    margin: $product-img-padding + 8 0 4px;
+  }
+
+  &__code {
+    color: $gray-color;
+    font-size: $font-S;
+    margin: 0;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin: 8px 0 $product-img-padding;
+
+    &-txt {
+      color: $gray-color;
+      margin: 0;
+    }
+
+    &-sum {
+      color: $page-text-color;
+      font-size: 16px;
+      font-weight: 700;
+      margin: 0;
+      line-height: 18px;
+    }
+  }
+
+  &__button-buy {
+    background-color: $button-color;
+    text-decoration: none;
+    color: $cart-bg-color;
+    text-transform: uppercase;
+    font-size: 14px;
+    font-weight: 700;
+    display: inline-block;
+    width: 166px;
+    height: $button-buy-height;
+    border-radius: 5px;
+    text-align: center;
+    line-height: $button-buy-height;
+    border: 1px solid $button-color;
+    box-sizing: border-box;
+
+    &:hover {
+      color: $button-color;
+      background-color: $cart-bg-color;
+    }
+  }
```

Comment:
Consider adding a transition effect to the button for a smoother color change. For example, `transition: background-color 0.5s, color 0.5s;`. This will make the color change more gradual and visually appealing.